ztools/x509/json.go

```diff
@@ -260,6 +260,21 @@ func (c *Certificate) MarshalJSON() ([]byte, error) {
 			jc.Names = append(jc.Names, name)
 		}
 	}
+	
+	for _, name := range c.URIs {
```

Comment:
Question for @zakird and @dadrian: Do we need to handle urls with *'s in them? If so this code won't work.